src/scripts/devhub.zig

```diff
@@ -129,6 +129,28 @@ fn devhub_metrics(shell: *Shell, cli_args: CLIArgs) !void {
         "us",
     );
 
+    const ci_pipeline_duration_s = blk: {
+        const times_gh = try shell.exec_stdout("gh run list -c {sha} -e merge_group " ++
+            "--json startedAt,updatedAt -L 1 --template {template}", .{
+            .sha = cli_args.sha,
+            .template = "{{range .}}{{.startedAt}} {{.updatedAt}}{{end}}",
+        });
+        var times_iterator = std.mem.split(u8, times_gh, " ");
+        const iso8601_started_at = times_iterator.next() orelse break :blk null;
+        const iso8601_updated_at = times_iterator.next() orelse break :blk null;
```

Comment:
```
        const iso8601_started_at, const iso8601_updated_at =
            (stdx.cut(times_gh, " ") orelse break :blk null).unpack();
```

Reply:
I always forget about our `stdx.*` utils, and that Zig now has destructuring. (perhaps suppressing JS memories...)